packages/foundations/scss/icon/_icons.placeholder.scss

```
@@ -1,12 +1,14 @@
 @use "icons.variables" as *;
 @use "../helpers/functions" as *;
+@use "../variables.global" as *;
 
 %icon {
 	// Hiding icon from screenreaders
 	-webkit-alt: "";
 	alt: "";
 
 	content: var(--icon-glyph);
+	color: $db-colors-neutral-on-bg;
```

```suggestion
	color: inherit;
```

In most cases, I could think about that this would mean a color inheritance instead.

---

fixed